packages/devtools_app/lib/src/screens/memory/shared/heap/heap.dart

```diff
@@ -215,6 +215,9 @@ class ObjectSet extends ObjectSetStats {
   static ObjectSet empty = ObjectSet()..seal();
 
   final objectsByCodes = <IdentityHashCode, AdaptedHeapObject>{};
+
+  /// Subset of objects which size is not included bu the method [countInstance]
+  /// into retained size of this set.
```

Comment:
docs nit: 
```
/// Subset of objects that are excluded from the retained size calculation for this set.
///
/// See [countInstance]
```

also, maybe rename to `objectsExcludedFromRetainedSize` to make this more self documenting